lib/internal/errors.js

```diff
@@ -79,9 +79,19 @@ module.exports = exports = {
 // Any error code added here should also be added to the documentation
 //
 // Note: Please try to keep these in alphabetical order
+E('ERR_ARG_NOT_ITERABLE', '%s must be iterable');
 E('ERR_ASSERTION', (msg) => msg);
 E('ERR_INVALID_ARG_TYPE', invalidArgType);
 E('ERR_INVALID_CALLBACK', 'callback must be a function');
+E('ERR_INVALID_FILE_URL_HOST', 'File URL host must %s');
+E('ERR_INVALID_FILE_URL_PATH', 'File URL path must %s');
+E('ERR_INVALID_THIS', 'Value of this must be of type %s');
```

Comment:
Maybe surround `this` with quotes or backticks? The original error message uses backticks.